osis-core/src/test/java/com/scality/osis/service/impl/ScalityOsisServiceTest.java

```diff
@@ -182,7 +183,7 @@ public void testCreateTenant500(){
 
         when(vaultAdminMock.createAccount(any(CreateAccountRequestDTO.class)))
                 .thenAnswer((Answer<CreateAccountResponseDTO>) invocation -> {
-                    throw new VaultServiceException(500, "EntityAlreadyExists");
+                    throw new VaultServiceException(HttpStatus.INTERNAL_SERVER_ERROR, "EntityAlreadyExists");
```

Comment:
Internal server error is due to the fault of the server.
Entry already exists is different.
I am confused why we have mix of both?

Here is our exhaustive list for reference: https://github.com/scality/Arsenal/blob/development/8.1/errors/arsenalErrors.json
Similar is followed by AWS.

Reply:
EntryAlreadyExists is 409
https://github.com/scality/Arsenal/blob/development/8.1/errors/arsenalErrors.json#L394

Reply:
I will change this anyways in the end, OSIS can return only 201 or 400 for `Create Account`